app/controllers/applications_controller.rb

```diff
@@ -16,14 +14,15 @@ def create
     if application.save
       redirect_to "/applications/#{application.id}"
     else
-      redirect_to "/applications/new", alert: "Please fill in answers to all prompts"
+      flash[:notice] = "Please fill in answers to all prompts"
```

Comment:
Cool, I think the way I had it was shorthand to write the flash notation, but I do like this way!

Reply:
Yeah, I could tell from the rails s that you're way worked perfectly well. I just jumped ahead to the lesson and slides on sad path and flash messages Tuesday and that's the notation they use, so I figured I'd just rewrite it that way for now because they might be scanning for it when they look at our project